deploifai/context.py

```diff
@@ -147,7 +147,7 @@ def project_found(f):
     @pass_context
     def wrapper(click_context, *args, **kwargs):
         deploifai_context = click_context.find_object(DeploifaiContextObj)
-        if deploifai_context.local_config is not None:
+        if deploifai_context.local_config["PROJECT"]["id"] is not None:
```

Comment:
I think this might throw a `KeyError` when trying to access a key that doesn't exist in the dictionary. So maybe check using `if 'id' in ...`